components/navbar/StatsDropdown.vue

```diff
@@ -1,83 +1,124 @@
+          </b-navbar-item>
+          <b-navbar-item
+            data-cy="series-insight"
+            to="/series-insight"
+            tag="nuxt-link">
+            {{ $t('series.label') }}
+          </b-navbar-item>
+        </template>
+        <template v-if="chain === 'rmrk'">
+          <b-navbar-item data-cy="spotlight" to="/spotlight" tag="nuxt-link">
+            {{ $t('spotlight.page') }}
+          </b-navbar-item>
+          <b-navbar-item
+            data-cy="series-insight"
+            to="/series-insight"
+            tag="nuxt-link">
+            {{ $t('series.label') }}
+          </b-navbar-item>
+          <b-navbar-item data-cy="sales" to="/sales" tag="nuxt-link">
+            {{ $t('sales.page') }}
+          </b-navbar-item>
+          <b-navbar-item data-cy="hot" to="/hot" tag="nuxt-link">
+            {{ $t('hot.label') }}
+          </b-navbar-item>
+        </template>
       </template>
-      <b-dropdown-item has-link aria-role="menu-item">
-        <nuxt-link data-cy="spotlight" to="/spotlight">
-          {{ $t('spotlight.page') }}
-        </nuxt-link>
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="series-insight" to="/series-insight">
-          Series</nuxt-link
-        >
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="sales" to="/sales"> Sales</nuxt-link>
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="hot" to="/hot"> Hot</nuxt-link>
-      </b-dropdown-item>
-    </b-dropdown>
+    </MobileExpandableSection>
   </div>
 </template>
-<script lang="ts">
-import { Component, Prop, mixins } from 'nuxt-property-decorator'
-import { getChainTestList } from '~/utils/constants'
-import PrefixMixin from '@/utils/mixins/prefixMixin'
-import AuthMixin from '~~/utils/mixins/authMixin'
+<script lang="ts" setup>
+import { isMobileDevice } from '~~/utils/extension'
 
-@Component({})
-export default class NavbarCreate extends mixins(PrefixMixin, AuthMixin) {
-  @Prop({ type: String }) chain!: string
+const MobileExpandableSection = defineAsyncComponent(
+  () => import('@/components/navbar/MobileExpandableSection.vue')
+)
+const { accountId } = useAuth()
 
-  get options() {
-    const availableUrlPrefixes = this.$store.getters['availableUrlPrefixes']
+const { urlPrefix } = usePrefix()
+const props = defineProps<{
+  chain?: string
+}>()
 
-    if (!this.$config.dev) {
-      return availableUrlPrefixes.filter(
-        (urlPrefix) => !getChainTestList().includes(urlPrefix.value as string)
-      )
-    }
-    return availableUrlPrefixes
-  }
-}
+const isMobile = window.innerWidth < 1024 ? true : isMobileDevice
+
+const offersUrl = `${
+  accountId.value
+    ? `/${urlPrefix.value}/offers?target=${accountId.value}`
+    : `/${urlPrefix.value}/offers`
+}`
+
+const statsUrl = `/${urlPrefix.value}/stats`
+
+const showSnekBsxOptions = props.chain === 'bsx' || props.chain === 'snek'
```

Comment:
These lines should convert to some getter funtion by using `computed()`.

Reply:
fixedd, thanks

Reply:
> > switch to bsx, then the create menu item dose not change correctly.
> 
> The bug is still there. Could you pls check it? @prachi00

my bad, the check was removed :/ , works now, sorry about that